addons/mail/static/src/components/notification_group/notification_group.js

```diff
@@ -67,7 +67,7 @@ class NotificationGroup extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickFailureDiscard(ev) {
+    _onClickMarkAsRead(ev) {
```

Comment:
I understand this is due to bad identity of menu but I'm not not sure if I like this rename.

I would prefer if `o_NotificationGroup_markAsRead` and ref `markAsRead` were renamed to failure discard.

Reply:
Thing is scss uses shared mixin too with all notification list item, and this mixin has "mark as read" in its name. Also visually all notification items display a check icon, so it makes sense for them to have a relatively same name.

That's why I would prefer keeping "mark as read" for all of them. At least, this is the most reasonable solution right now IMO.